### ISSUE TITLE
Add Keyboard Navigation to '@ Mentions' Example

### ISSUE THREAD

**Issue by 0ffwolf** [ianstormtaylor/slate#2613]
#### Do you want to request a _feature_ or report a _bug_?

Feature? Just a little confused on how to add keyboard navigation to the current implementation of the mentions example (i.e. '@ Lu ' then pressing down should let you choose between available users)

Spoke to @ericedem and he stated that I should make an issue to track, thanks!

#### What's the current behavior?

Keyboard navigation is not currently handled

#### What's the expected behavior?

Keyboard navigation should be handled

**Comment by 0ffwolf** [comment]
https://github.com/ianstormtaylor/slate/issues/2609
